src/main/java/org/isf/medicalinventory/gui/InventoryBrowser.java

```diff
@@ -393,9 +412,27 @@ class InventoryBrowsingModel extends DefaultTableModel {
 		private static final long serialVersionUID = 1L;
 
 		public InventoryBrowsingModel() {
+			inventoryList = new ArrayList<>();
+			String state = stateComboBox.getSelectedIndex() > 0 ? stateComboBox.getSelectedItem().toString().toLowerCase(): null;
+			String type = InventoryType.principal.toString();
+			try {
+				inventoryList = medicalInventoryManager.getMedicalInventoryByParams(dateFrom, dateTo, state, type);
+			} catch (OHServiceException e) {
+				OHServiceExceptionUtil.showMessages(e);
+			}	
 		}
-
+		
 		public InventoryBrowsingModel(int start_index, int page_size) {
```

Comment:
The project uses camelCase variables and not those with underscores:  so `startIndex` instead of `start_index`, etc.